src/main/java/com/spotify/futures/Combined.java

```diff
@@ -0,0 +1,49 @@
+/*
+ * Copyright (c) 2019 Spotify AB
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License"); you may not
+ * use this file except in compliance with the License. You may obtain a copy of
+ * the License at
+ *
+ * http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS, WITHOUT
+ * WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied. See the
+ * License for the specific language governing permissions and limitations under
+ * the License.
+ */
+package com.spotify.futures;
+
+import java.util.IdentityHashMap;
+import java.util.List;
+import java.util.concurrent.CompletionStage;
+
+public class Combined {
```

Comment:
Nit: Consider naming this `CombinedFutures` or something to reduce the likelihood of collisions.

Also nit: Consider declaring this class as `final` since it is not meant to be extended.

Reply:
Done!